### PR TITLE
Adding support for deauthorizeToken

### PR DESCRIPTION
**Pull request by EllenPorter**

### CODE REVIEW

File: org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js [hunk 1]
```diff
@@ -56,6 +56,24 @@ class Translator {
...
+     */
+    deauthorizeToken(authInfo) {
+        
+        var options = {
+            url: 'https://api.home.nest.com/oauth2/access_tokens/' + this._authTokens['access'].token,
+            method: 'DELETE'
+        };
+
+        return request(options).then((body) => {
+            return true;
```
Comment (kurstory) [inline org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js]: If this request fails, can we have it return false?

Comment (EllenPorter) [inline org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js]: Yup, good idea, fixed.

File: org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js [hunk 2]
```diff
@@ -178,6 +178,17 @@ class Translator {
     }
 
     /**
+     * Adding for consistency across hubs
+     */
+    deauthorizeToken(authInfo) {
+        if (!authInfo) {
```
Comment (kurstory) [inline org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js]: Why is this distinction only in Wink and not the other providers?

Comment (EllenPorter) [inline org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js]: The Travis check marked the unused variable as an error - looking now to see why it was an error not warning like the other providers. If we implement deauthorize in the other providers we might need the authInfo, so I'd like to keep the variable being passed in.

**Pull request merged**